src/app/components/Account/index.tsx

```diff
@@ -19,11 +20,13 @@ export const addressToNumber = (address: string) => {
 type AccountProps = {
   // switch to Emerald when endpoint is ready
```

Comment:
This is not something we have discussed earlier, but I prefer adding `TODO` to code comments that explains something that we need to do later.